traffic_control/clients/python/pylint.rc

```diff
@@ -429,6 +429,12 @@ max-branches=15
 # Maximum number of statements in function / method body.
 max-statements=80
 
+# Maximum number of locals for function / method body
+max-locals=25
+
+# Maximum number of arguments for function / method
+max-args=10
+
```

Comment:
Why add these? Was there a default value that was causing your code to not pass the linter?

Reply:
yes, There's a default value that was causing my code to not pass the linter.
Error: Too many local variables (18/15) (too-many-locals), 
So i've added these max-locals in pylint.rc . I've removed these max-args as there is no use to have it in pylint.rc file